source/guide_restic.rst

```diff
@@ -0,0 +1,211 @@
+
+  * https://github.com/restic/restic/blob/master/LICENSE
+
+
+Prerequisites
+=============
+
+We're using :manual:`Go <lang-golang>` version go1.12.5 to compile the software.
+
+::
+
+ [isabell@stardust ~]$ go version
+ go version go1.12.5 linux/amd64
+ [isabell@stardust ~]$
+
+If you want to back up to a remote server, you should set up a bucket a or user first. Restic currently supports backup with:
+
+  - `Local directory <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#local>`__
+  - `sftp server (via SSH) <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#sftp>`__
+  - `HTTP REST server <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#rest-server>`__ (`protocol <doc/100_references.rst#rest-backend>`__ `rest-server <https://github.com/restic/rest-server>`__)
+  - `AWS S3 <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#amazon-s3>`__ (either from Amazon or using the `Minio <https://minio.io>`__ server)
+  - `OpenStack Swift <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#openstack-swift>`__
+  - `BackBlaze B2 <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#backblaze-b2>`__
+  - `Microsoft Azure Blob Storage <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#microsoft-azure-blob-storage>`__
+  - `Google Cloud Storage <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#google-cloud-storage>`__
+  - And many other services via the `rclone <https://rclone.org>`__ `Backend <https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#other-services-via-rclone>`__
+
+Installation
+============
+
+``cd`` to your ``/$HOME`` home directory before fetching Restic.
+
+We're building restic from source with the following steps:
+
+::
+
+ [isabell@stardust ~]$ git clone https://github.com/restic/restic.git
+ [...]
+ [isabell@stardust ~]$ cd restic
+ [isabell@stardust restic]$ go run build.go
+ [...]
+ [isabell@stardust restic]$
+
+
+Configuration
+=============
+
+Login details
+-------------
+Now it's time to get you login/API details from you remote server. This differs from service to service, but usually boils down to a key id (username) and a key (password).
+
+Setup repository
+----------------
+.. note:: The service or protocol used, needs to be specified, when communicating with a repository. Please read in the restic documentation: https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html
+
+.. warning:: Restic repositories are encrypted, and losing the password means losing access to all the files. So please save the password somewhere safe.
+
+Here we're initializing a repository on a Backblaze B2 bucket
+
+::
+
+  [isabell@stardust ~]$ export B2_ACCOUNT_ID=<MY_APPLICATION_KEY_ID>
+  [isabell@stardust ~]$ export B2_ACCOUNT_KEY=<MY_APPLICATION_KEY>
+  [isabell@stardust ~]$ ~/restic/restic -r b2:bucketname:path/to/repo init
```

Comment:
Please drop the binary in `~/bin` so it can be used without the full path.

Reply:
Fix in commit 2ff100c